html/js/main.js

```diff
@@ -22,4 +22,19 @@
         return false;
     });
 
+    var playerButtons = document.getElementsByClassName("button-player");
+    for (var i = 0; i < playerButtons.length; i++) {
+        playerButtons[i].onclick = GTE.tools.buttonPlayerHandler(playerButtons[i].getAttribute("player"));
```

Comment:
For consistency with the rest of the file, why don´t you use _addEventListener_ here too?